ssp

```diff
@@ -8538,7 +8540,7 @@ END_OF_TEMPLATE
         if ( $pid == 0 ) {
             close $out_r;
 
-            open STDIN,  '</dev/null';
+            open STDIN,  '<',  '/dev/null';
```

Comment:
Why this change - curiosity sake.

Reply:
2-arg open is generally frowned upon these days when not opening sockets.  Mostly because you can get in trouble with files that have operation characters in them (e.g. >,<).

Reply:
perlcritic also forbids it (for the reason George mentioned) so 'make test' failed with:
`[InputOutput::ProhibitTwoArgOpen] Line 8543 Column 13 - Two-argument "open" used - See page 207 of PBP. (Severity: 5)`